pkg/controller/controller_instance_test.go

```diff
@@ -643,6 +648,49 @@ func TestReconcileInstanceDelete(t *testing.T) {
 	}
 }
 
+func TestReconcileInstanceDeleteDoesNotInvokeBroker(t *testing.T) {
```

Comment:
Can you put a comment above this `func` to explain basically what it's testing? A sentence or 2 is enough